cli/tools/registry/diagnostics.rs

```diff
@@ -258,7 +259,32 @@ impl Diagnostic for PublishDiagnostic {
   }
 
   fn snippet_fixed(&self) -> Option<DiagnosticSnippet<'_>> {
-    None
+    match &self {
+      PublishDiagnostic::InvalidExternalImport { imported, .. } => {
+        match super::api::get_jsr_alternative(imported) {
+          Some(replacement) => {
+            let replacement = SourceTextInfo::new(replacement.into());
+            let start = replacement.line_start(0);
+            let end = replacement.line_end(0);
+            Some(DiagnosticSnippet {
+              source: DiagnosticSnippetSource::SourceTextInfo(Cow::Owned(
+                replacement,
+              )),
```

Comment:
Actually, all you need to do this this:

```suggestion
              source: Cow::Owned(replacement),
```